package.json

```diff
@@ -2,7 +2,7 @@
   "name": "roblox-win-installer-action",
   "scripts": {
     "build": "tsc",
-    "pack": "ncc build src/main.ts"
+    "pack": "ncc build src/index.ts"
```

Comment:
I may not understand the usage of this well enough. I figured it was intended to be `index.ts` rather than `main.ts`. Please let me know if I'm misunderstanding.